rasterio/coords.py

```diff
@@ -9,18 +9,11 @@
     .. code::
 
         BoundingBox(left, bottom, right, top)
-
-    Attributes
-    ----------
-    left :
-        Left coordinate
-    bottom :
-        Bottom coordinate
-    right :
-        Right coordinate
-    top :
-        Top coordinate
     """
+BoundingBox.left.__doc__ = "Left coordinate"
```

Comment:
Might be good to add a comment so others don't change it back in the future and break it.